retrodep/vendored.go

```diff
@@ -343,6 +343,13 @@ func (src GoSource) DescribeProject(
 		topver = top.Ver
 	}
 
+	ref = &Reference{
+		TopPkg: toppkg,
+		TopVer: topver,
+		Pkg:    project.Root,
+		Repo:   project.Repo,
+	}
+
```

Comment:
Maybe now we can just update this `ref` to set the other fields, in the other places we currently create a new one?

Reply:
Done. Thanks for the suggestion.